desktop.css

```diff
@@ -30,12 +30,13 @@
     justify-content: space-between;
     float: right;
     flex-direction: column;
-    font-family: sans-serif;
-    font-size: 18px;
     position: relative;
     bottom: 70px;
     margin: 30px 40px;
     align-items: center;
+    font-family: 'Inter', sans-serif;
+    font-weight: 600;
+    font-size: 15px;
   }
 
   .header-menu {
```

Comment:
[OPTIONAL] Please note that the mobile design should still work after making styles for desktop. Yours currently deviates from the figma design. 
This is a critical requirement for this project and I would recommend you fix it. I only highlighted is as optional since previous reviewers missed it but note that on the next milestone, you will be penalised.
## Your design
<img width="451" alt="Screenshot 2022-10-01 at 18 42 53" src="https://user-images.githubusercontent.com/48546992/193417773-e4426a2b-f63d-43bc-9bec-26302322ee95.png">